frontend/leads-list/table/lead-services-cell.component.tsx

```diff
@@ -0,0 +1,204 @@
+            </div>
+          )}
+          {nutritionServices.length > 0 && (
+            <div className="icon-container">
+              <img
+                className="program-icon"
+                alt="nutrition-icon"
+                src={nutritionIcon}
+                title="Nutrition / CRYS / SNAP"
+              />
+              <span>( {nutritionServices.length} )</span>
+            </div>
+          )}
+          {immigrationServices.length > 0 && (
+            <div className="icon-container">
+              <img
+                className="program-icon"
+                alt="immigration-icon"
+                src={immigrationIcon}
+                title="Immigration"
+              />
+              <span>( {immigrationServices.length} )</span>
+            </div>
+          )}
+          {financialServices.length > 0 && (
+            <div className="icon-container">
+              <img
+                className="program-icon"
+                alt="financial-icon"
+                src={financeIcon}
+                title="Financial Education / Coaching"
+              />
+              <span>( {financialServices.length} )</span>
+            </div>
+          )}
+          {workersRightsServices.length > 0 && (
+            <div className="icon-container">
+              <img
+                className="program-icon"
+                alt="workers-rights-icon"
+                src={workerRightsIcon}
+                title="Workers' Rights"
+              />
+              <span>( {workersRightsServices.length} )</span>
+            </div>
+          )}
+          {familySupportServices.length > 0 && (
+            <div className="icon-container">
+              <img
+                className="program-icon"
+                alt="family-services-icon"
+                src={familySupportIcon}
+                title="Family Support"
+              />
+              <span>( {familySupportServices.length} )</span>
+            </div>
+          )}
+          {communityEngagementServices.length > 0 && (
+            <div className="icon-container">
+              <img
+                className="program-icon"
+                alt="community-engagement-icon"
+                src={communityEngagementIcon}
+                title="Community Engagement and Organizing"
+              />
+              <span>( {communityEngagementServices.length} )</span>
+            </div>
+          )}
+          {focusGroupServices.length > 0 && (
+            <div className="icon-container">
+              <img
+                className="program-icon"
+                alt="focus-groups-icon"
+                src={focusGroupsIcon}
+                title="Focus Group"
+              />
+              <span>( {focusGroupServices.length} )</span>
+            </div>
+          )}
+        </div>
+      ) : (
+        <p>No services selected</p>
+      )}
+    </>
+  );
+}
+
+const css = `
+	& .programs-cell {
```

Comment:
Non-blocking: I looked at the [navbars component](https://github.com/JustUtahCoders/comunidades-unidas-internal/blob/fe1576b894c3a51ee4e483b629b0c43fe5864c6c/frontend/navbar/navbars.component.tsx), and it looks like the indentation of the css object should be two spaces instead of 6.
Could you make sure the formatting of this string is the same as the rest of the repository?